elasticgraph-warehouse/README.md

```diff
@@ -40,4 +38,109 @@ index 4a5ef1e..5c16c2b 100644
 
 ```
 
-TODO: Finish
+Next, update your `Rakefile` so that `ElasticGraph::Warehouse::SchemaDefinition::APIExtension` is
+used as one of the `extension_modules`:
```

Comment:
```suggestion
used as one of the `schema_definition_extension_modules`:
```

...since the attribute is called `schema_definition_extension_modules`, not `extension_modules`.